CONTRIBUTING.rst

```diff
@@ -82,17 +82,18 @@ If you haven't written or modified any tests, tests can be run just like:
 
 .. code-block:: bash
 
-    pytest
+    pytest <test folder name>
```

Comment:
Why did you change this? If you are in the root directory of the repository, it should work out of the box without specifying the directory.

Reply:
At first, that didn't work for me, it didn't execute any tests at all. Now that you mention it, I tried it without a default, and now it does work. Nonetheless, I think it is a useful addition, since it explains how to switch between the tests in the 'test' and 'tests_new' folders.